### PR TITLE
df-bp-homepage

### PR DESCRIPTION
**Pull request by chachi69**
Changes received from Beth Perry on 8/3/18.

### CODE REVIEW

**Comment by chachi69** [comment]
[Homepage-combined_BP.pdf](https://github.com/frtib/tsp-redesign/files/2260639/Homepage-combined_BP.pdf)

File: _sass/components/_home.scss [hunk 1]
```diff
@@ -30,15 +30,15 @@
   .usa-width-two-thirds {
     padding-right: $spacing-md-small;
   }
-  .usa-width-one-third {
+  .usa-width-one-third, .usa-width-one-fourth {
```
Comment (line47) [inline _sass/components/_home.scss]: I think this empty style block can be removed

**Comment by line47** [comment]
With tis new layout I think some updated design considerations are needed.

**The titles on multiple lines looks a bit like a ragged edge** 
screen shot 2018-08-06 at 9 44 13 am

**Maybe the L funds subtitle can be styled a bitt differently?** 
screen shot 2018-08-06 at 9 44 33 am

☝️ @Lapedra - what do you think?

**Comment by chachi69** [comment]
@Lapedra I probably wasn't clear about `Why the TSP?` I meant roll back the change (i.e., delete and restore it back to how it was).

**Comment by chachi69** [comment]
Output of my meeting with Beth.

image

**Comment by chachi69** [comment]
Need to migrate to AWS in order to facilitate wider team discussion.

**Pull request merged**